modules/virt-preventing-workload-updates-during-eus-update.adoc

```diff
@@ -0,0 +1,201 @@
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Available"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Progressing"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T16:39:11Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "False",
+    "type": "Degraded"
+  },
+  {
+    "lastTransitionTime": "2022-12-09T20:30:10Z",
+    "message": "Reconcile completed successfully",
+    "observedGeneration": 3,
+    "reason": "ReconcileCompleted",
+    "status": "True",
+    "type": "Upgradeable" <1>
+  }
+]
+----
+====
+<1> The {VirtProductName} Operator has the `Upgradeable` status.
+
+. Manually update your cluster from the source EUS version to the next minor version of {product-title}:
++
+[source,terminal]
++
+----
+$ oc adm upgrade
+----
++
+.Verification
+* Check the current version by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+[NOTE]
+====
+Updating {product-title} to the next version is a prerequisite for updating {VirtProductName}. For more details, refer to the "Updating clusters" section of the {product-title} documentation.
+====
+
+. With the default *Automatic* approval strategy, {VirtProductName} automatically updates to the corresponding version after you update {product-title}. Monitor the {VirtProductName} update by running the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-cnv
+----
+
+. Continue to monitor as {VirtProductName} updates to the z-stream versions that are available for the non-EUS version.
+
+. Confirm that {VirtProductName} successfully updated to the latest z-stream release of the non-EUS version by running the following command:
++
+[source,terminal]
+----
+$ oc get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.versions"
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+[
+  {
+    "name": "operator",
+    "version": "{HCOVersion}"
+  }
+]
+----
+
+. Wait until the `HyperConverged` Operator has the `Upgradeable` status before you perform the next update. Enter the following command and monitor the output:
++
+[source,terminal]
+----
+$ oc get hco kubevirt-hyperconverged -n openshift-cnv -o json | jq ".status.conditions"
+----
+
+. Update to the target EUS version.
+.. Update {product-title} to the target EUS version. Confirm that the update succeeded by checking the cluster version:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
+.. After {product-title} updates, {VirtProductName} automatically updates to the target EUS version. Monitor the update by running the following command:
```

Comment:
`automatically updates to the target EUS version` this is assuming automatic approval of install plan.